src/geometry/dual_quaternion.rs

```diff
@@ -78,3 +81,46 @@ where
         }
     }
 }
+
+#[cfg(feature = "serde-serialize")]
+impl<N: SimdRealField> Serialize for DualQuaternion<N>
+where
+    N: Serialize,
+{
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        [
```

Comment:
Wasn't sure how to use a `VectorN` similar to how quaternions serialization works so I opted for this.

Let me know if this can be improved and I'll make the change(s).

Reply:
If your rebase your PR on `dev`, you should be able to `derive(Serialize, Deserialize)` directly on `DualQuaternions`.